CMakeLists.txt

```diff
@@ -301,14 +301,14 @@ configure_file(mpi-cxx.pc.in mpi-cxx.pc @ONLY)
 configure_file(mpi-fort.pc.in mpi-fort.pc @ONLY)
 install(
```

Comment:
Can you add a brief comment here stating why `CMAKE_CURRENT_BINARY_DIR` is needed, and how it is different from `CMAKE_BINARY_DIR`? Otherwise I might forget at some time in the future.